packages/ember-simple-auth/addon/internal-session.js

```diff
@@ -21,6 +20,12 @@ export default ObjectProxy.extend(Evented, {
     this._bindToStoreEvents();
   },
 
+  store: computed(function() {
+    const owner = getOwner(this);
+    const store = Ember.testing ? 'session-store:test' : 'session-store:application';
```

Comment:
The global `Ember` is deprecated

Reply:
That's a very good point but AFAIK there are no good alternatives other than reading environment variable from config.
Also, we're using Ember.testing elsewhere so I think it's ok-ish, to have it here but once it becomes a real problem, it can be dealt with later.

Reply:
Yes, I think that's ok since indeed it's used elsewhere as well – should be cleaned up but not as part of this.